src/domains/profile/pages/CreateProfile/CreateProfile.test.tsx

```diff
@@ -72,22 +62,27 @@ describe("CreateProfile", () => {
 		env.reset({ coins: { ARK }, httpClient, storage: new StubStorage() });
 	});
 
-	beforeEach(() => {
-		browserAccessMock = vi
-			.spyOn(browserAccess, "fileOpen")
-			.mockResolvedValue(new File([], "picture.png", { type: "image/png" }));
-	});
+	it("should render", async () => {
+		await renderComponent();
 
-	afterEach(() => {
-		browserAccessMock.mockRestore();
+		userEvent.click(screen.getByText("Back"));
```

Comment:
Not a must, but do you think we can add the `await` before every user event for the tests? It will help to prevent the tests from breaking after we merge with the test refactor branch.

Reply:
![image](https://github.com/user-attachments/assets/0a4aafac-cf99-4683-87e1-83a47675cbe5)

That should be handled when we merge tests.